vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -123,11 +123,13 @@ $global:map = @{
         "https://acs-mirror.azureedge.net/kubernetes/v1.19.11-hotfix.20210526/windowszip/v1.19.11-hotfix.20210526-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.19.12/windowszip/v1.19.12-1int.zip",
         "https://acs-mirror.azureedge.net/kubernetes/v1.19.13/windowszip/v1.19.13-1int.zip",
+        "https://acs-mirror.azureedge.net/kubernetes/v1.19.13/windowszip/v1.19.13-hotfix.20210830-1int.zip",
```

Comment:
The URLs should be 
https://acs-mirror.azureedge.net/kubernetes/v1.19.13-hotfix.20210830/windowszip/v1.19.13-hotfix.20210830-1int.zip
https://acs-mirror.azureedge.net/kubernetes/v1.20.9-hotfix.20210830/windowszip/v1.20.9-hotfix.20210830-1int.zip
It seems like that all zip packages are still not available now.

Reply:
Thanks. I've updated the links. Are the links for both Windows and Linux versions automatically generated through the build pipeline for the hotfix? If that is the case, it is likely because the pipeline did not finish publishing yet.

Reply:
They will be available after upstream team approve it

Reply:
should be published now

Reply:
Thanks Abel and Ace!